cmd/ocm-backplane/cloud/common.go

```diff
@@ -96,16 +99,16 @@ func getIsolatedCredentials(clusterID string, ocmToken *string) (aws.Credentials
 		Credentials: NewStaticCredentialsProvider(seedCredentials.AccessKeyID, seedCredentials.SecretAccessKey, seedCredentials.SessionToken),
 	})
 
-	targetCredentials, err := AssumeRoleSequence(email, seedClient, roleAssumeSequence, bpConfig.ProxyURL, awsutil.DefaultSTSClientProviderFunc)
+	targetCredentials, err := AssumeRoleSequence(email, seedClient, roleAssumeSequence, cfg.BackplaneConfiguration.ProxyURL, awsutil.DefaultSTSClientProviderFunc)
 	if err != nil {
 		return aws.Credentials{}, fmt.Errorf("failed to assume role sequence: %w", err)
 	}
 	return targetCredentials, nil
 }
 
-func isIsolatedBackplaneAccess(cluster *cmv1.Cluster) (bool, error) {
+func isIsolatedBackplaneAccess(cluster *cmv1.Cluster, ocmConnection *ocmsdk.Connection) (bool, error) {
```

Comment:
```suggestion
func (cfg *QueryConfig) isIsolatedBackplaneAccess(cluster *cmv1.Cluster) (bool, error) {
```

Any reason to not make this (and kind of all the other functions) methods off of `QueryConfig`? To me, that's one of the main benefits of defining a `QueryConfig` in the first place - you don't have to worry about passing the right arguments?

I could be convinced otherwise though, currently I'm thinking as long as the public/external helper functions are methods that's good enough for me (e.g. skip this one and refactor `GetAWSV2Config` to be a method).

Reply:
Agree that it makes sense, I was being a bit too careful with refactoring I suppose.

The following are now members of `QueryConfig` (and have been moved to `common.go`: 
- `getCloudCredentials` -> public 
- `getCloudCredentialsFromBackplaneAPI`
- `getCloudConsole` -> public

I also refactored `GetCloudConsole` the same way `GetCloudCredentials` has been refactored to enable calling it from `console.go`.

**After these changes I tested the following:**
Old flow:
- ocm backplane cloud credentials
- ocm backplane cloud console

Non STS clusters:
- ocm backplane cloud credentials
- ocm backplane cloud console

New flow:
- ocm backplane cloud credentials
- ocm backplane cloud console